app/controllers/simulations_controller.rb

```diff
@@ -59,8 +59,119 @@ def destroy_component
     redirect_to :action => :index
   end
 
+  #TODO errors also in Polish
```

Comment:
Nie, chodzi na razie tylko o i18n :) - używanie funkcji I18n.t
Wersja PL, jeśli w ogóle kiedykolwiek będzie, to będzie zadanie przetłumaczenia całego Scalarma.